services/web/server/src/simcore_service_webserver/exporter/_formatter/_sds.py

```diff
@@ -160,7 +160,14 @@ async def create_sds_directory(
     # assemble params here
     dataset_description_params = parse_obj_as(
         DatasetDescriptionParams,
-        {"name": project_data["name"], "description": project_data["description"]},
+        {
+            "name": project_data["name"],
+            "description": (
+                ""
```

Comment:
THOUGHT Empty strings are not good sentinel values for ` DatasetDescriptionParams`. I would set a default otherwise you will end up in the never-ending story of validators that convert empty stuff into None etc ... (checkout `common_validators` )

Reply:
Since this is an export of the data, if it's None empty string is OK for the people whom will be using it.